src/pages/home/uploadSong/UploadSong.tsx

```diff
@@ -44,6 +44,15 @@ const UploadSong: FunctionComponent = () => {
     genres: [],
     moods: [],
     description: "",
+    language: undefined,
+    copyrights: undefined,
+    barcodeType: undefined,
+    barcodeNumber: undefined,
+    isrc: undefined,
+    iswc: undefined,
+    ipis: undefined,
+    userIpi: "",
+    releaseDate: undefined,
```

Comment:
Added these as placeholders to note all the initialValues, they're listed as undefined so they aren't sent through to the endpoint if left empty. Is it useful to include them to note which values might be touched or should they be left off? Should we modify the other optional fields to undefined? The only one that will become prefilled is the userIpi.

Reply:
That was my thinking as well but leaning on the other end. Wanted the second opinion and it is redundant to have those values since when excluded they do still present as `undefined`. There are also other indicators to what fields/values are present. I will clean it up to removed the optional parameters.

Reply:
Just double checked a different scenario of just adding and then removing data from the optional Advanced Details page fields. When the fields are touched they end up as string values which doesn't play nice with the endpoint. We will need to do checks on values that are empty strings and replace them with undefined.